packages/itwinui-react/src/utils/components/InputFlexContainer.tsx

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-import React from 'react';
+import { forwardRef } from 'react';
```

Comment:
The styleguide is fine. This change was made by the codemod. I've fixed it in [`172351a`](https://github.com/iTwin/iTwinUI/pull/2102/commits/172351aa3d157139b10d55c1cfdf4142331971d8).

The important thing to remember is that default import is discouraged because it will be removed in a future version of React. Namespace and named imports are both fine, but we prefer namespace imports because it makes it easier to use anything from `React` without changing imports.